resources/dicts/events/lifegen_events/murder.json

```diff
@@ -307,25 +308,27 @@
         "v_c has never been the sharpest claw in the clan, but their attitude only makes everything worse. Sure, they're an elder, but that isn't an excuse for v_c to constantly and consistently belittle and disregard you. One day, after hearing v_c talk about missing their days as a warrior, you tell them that you'd be more than happy to take them out on a walk through the territory so that they can relive their youth. v_c reluctantly agrees, and doesn't realize in time that you led them to a fox trap. You leave v_c for dead as you hear their choked sobs get quieter and quieter."
     ],
     "murder deputy kitten": [
-        "You encourage v_c to eat some shiny red berries you've found, saying that they're super tasty - better than freshkill even! Happily, they thank you for the snack. v_c is later found, lifeless, with deathberry juice dripping from their mouth.",
-        "You offer to take v_c onto a grand and exciting adventure out of the nursery. You help v_c sneak out and the two of you run into the woods. When you're far enough from camp, you suddenly speed up, leaving v_c behind as their startled cries call out your name from behind. You loop back to camp, alone, and no one asks where you've been.",
-        "You tell v_c to follow you, saying that you want to show them a shiny treasure you found recently. They happily agree and the two of you sneak out of camp. You lead v_c to a foxes' den, and they ask you what the smell is with a wrinkled nose. You tell them not to worry about it and push them forward, saying that the treasure is inside. With an anxious look at you, they slowly head in. You don't stay to listen to the foxes' feasting.",
+        "You encourage v_c to eat some shiny red berries you've found, saying that they're super tasty - better than freshkill even! Happily, they thank you for the snack. v_c is later found, lifeless, already having gone to StarClan.",
+        "You offer to take v_c onto a grand and exciting adventure out of the nursery. You help v_c sneak out and the two of you run into the woods. When you're far enough from camp, you suddenly speed up, leaving v_c behind. You loop back to camp, alone, and no one asks where you've been.",
+        "You tell v_c to follow you, saying that you want to show them a shiny treasure you found recently. They happily agree and the two of you sneak out of camp. You lead v_c to a foxes' den, and they ask you what the smell is with a wrinkled nose. You tell them not to worry about it and push them forward, saying that the treasure is inside. With an anxious look at you, they slowly head in, and never come back out.",
```

Comment:
this can be changed to match the previous one

"You tell v_c to follow you, saying that you want to show them a shiny treasure you found recently. They happily agree and the two of you sneak out of camp. You lead v_c to a foxes' den, and they never come back out."